schema.graphql

```diff
@@ -340,7 +341,11 @@ type Mutation {
     toolsQuota: QuotaInput!
   ): PrivateCloudRequest! @auth
 
-  privateCloudProjectDeleteRequest(projectId: ID!): PrivateCloudRequest! @auth
```

Comment:
these changes are not reflected on front part, that causes an error  ""Field "privateCloudProjectDeleteRequest" argument "licencePlate" of type "String!" is required, but it was not provided"